### ISSUE TITLE
[Servicing] Accessibility: no way to set `AccessibleName` for selected controls

### ISSUE THREAD

**Issue by vladimir-krestov** [dotnet/winforms#3504]
* .NET Core Version: 3.1

* Have you experienced this same bug with .NET Framework?: Yes
Original [Bug 1108206](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1108206)

**Problem description:**
If AccessibleName, AccessibleRole, AccessibleDescription is set to a custom value for the Label/GroupBox control, this does not affect the AccessibleName, AccessibleRole, AccessibleDescription in Accessibility tools. And default values are shown.

**Expected behavior:**
There should be an ability to set/change AccessibleName, AccessibleRole, AccessibleDescription for:
* [x] Label
* [x] GroupBox
* [x] ToolStrip, MenuStrip, StatusStrip, 
* [x] ToolStripItems, MenuItems, StatusStripItems
* [x] PropertyGrid
* [x] ProgressBar
* [x] ComboBox
* [x] DataGridView
* [x] Form

**Minimal repro:**
Create the app with the Label/GroupBox control. Specify custom AccessibleName, AccessibleRole, AccessibleDescription for the label/groupBox. Run the app and observe that the default AccessibleRole or default AccessibleName or default AccessibleDescription are shown in Inspect or Accessibility Insights.

**Comment by Tanya-Solyanik** [comment]
@vladimir-krestov, @SergeySmirnov-Akvelon  - could you please share how you generated the list of affected controls? 
Did you test all controls winforms controls that are available the toolbox in the classic designer in a 3.1 applicaiton?

**Comment by vladimir-krestov** [comment]
I created a test app, put all common controls from the Designer. Also, I added `ToolStrip/Menu/StatusStrip` items. I added custom values for accessible `Name`, `Role` and `Description` properties of all Controls and items. Then I checked every control on my form using Inspect. I checked `Name`, `LegacyIAccessible.Name`, `LegacyIAccessible.Role`, `LegacyIAccessible.Description` properties. 
Yes, we checked it in a 3.1 app.

**Comment by SergeySmirnov-Akvelon** [comment]
Accessible Name for Form control also was fixed in scope of this issue.

**Comment by Olina-Zhang** [comment]
@SergeySmirnov-Akvelon verified this bug in .Net Core September Update build: 3.1.108 by [VS VAL build](https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/269963) as PR [3600](https://github.com/dotnet/winforms/pull/3600) was merged, just AccessibleName of those controls can be customized, and AccessibleRole and AccessibleDescription cannot.  Do you have plan to fix AccessibleRole and AccessibleDescription in servicing 3.1 in this issue or open a new one to track?

**Comment by SergeySmirnov-Akvelon** [comment]
Hi @Olina-Zhang. We have #3712 and #3713 issues which are related to AccessibleRole. I think we will fix AccessibleRole and AccessibleDescription in scope of these issues.

**Comment by RussKie** [comment]
We must fix `AccessibleName` across the board for .NET 5.0, and `AccessibleRole` and `AccessibleDescription` can be fixed in .NET 6.0 timeframe.

**Comment by RussKie** [comment]
Closing as all referenced controls have been updated.
